### ISSUE TITLE
Flycheck displaying error when no error is reported by Eslint

### ISSUE THREAD

**Issue by bastos** [flycheck/flycheck#1356]
### What you did, and what you expected to happen instead

 I've opened my JSX file with no errors reported by eslint in command line and Flycheck displayed:

```    
1     warning         No JSCS configuration found.  Set `flycheck-jscsrc' for JSCS (javascript-jscs)
```

Running on command line I get

```
bastos@bastos ~/xyz $ eslint src/App.jsx
bastos@bastos ~/xyz $ echo $?
0
```

If I change the file to add an actual error, Flycheck works correctly, only displays that message when there is no error in the file.

### Whether and how you were able to [reproduce the issue in emacs -Q][emacsQ]

Only happens when using Flycheck.

### Your Flycheck setup from `M-x flycheck-verify-setup`

```
Syntax checkers for buffer App.jsx in rjsx-mode:

  javascript-eslint
    - may enable:  yes
    - executable:  Found at /usr/local/bin/eslint
    - config file: found

  javascript-jshint (disabled)
    - may enable:         yes
    - executable:         Found at /usr/local/bin/jshint
    - configuration file: Not found

  javascript-jscs
    - may enable:         yes
    - executable:         Found at /usr/local/bin/jscs
    - configuration file: Not found

  javascript-standard
    - may enable: Automatically disabled!
    - executable: Not found

The following checker is explicitly selected for this buffer:

  javascript-eslint
    - major mode:  `rjsx-mode' supported
    - may enable:  yes
    - executable:  Found at /usr/local/bin/eslint
    - config file: found

Flycheck Mode is enabled.  Use C-u C-c ! x to enable disabled
checkers.

--------------------

Flycheck version: 32snapshot (package: 20171026.610)
Emacs version:    25.3.3
System:           x86_64-apple-darwin13.4.0
Window system:    mac
```

### Your operating system

`macOS 10.12.6 (16G29)`

### Your Emacs version from `M-x emacs-version`

`25.3.3`

### Your Flycheck version from `M-x flycheck-version`

`32snapshot (package: 20171026.610)`

### .eslintrc.json

```json
{
  "settings": {
    "import/core-modules": [
      "react"
    ]
  },
  "extends": "airbnb",
  "plugins": [
    "react",
    "import"
  ]
}
```

### Screenshot

![boring](https://user-images.githubusercontent.com/11134/32425229-b840fe96-c298-11e7-98a0-19a1ba09e7a6.png)

**Comment by fmdkdd** [comment]
Thanks for the report.

The warning has `(javascript-jscs)` at the end, which indicates that the `javascript-jscs` checker produced it.  `javascript-eslint` is your default checker, and produces no warnings, so flycheck goes on to call the `:next-checker`, which is `javascript-jscs`.  You have `jscs`, but no config file, as indicated by `flycheck-verify-setup`. 

You can either provide a config file for `jscs`, or disable that checker if you don't want it to run.

**Comment by cpitclaudel** [comment]
Closing this as it looks like @fmdkdd posted a pretty complete answer :) @bastos please reopen if anything's unclear.
